backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxGenerator.java

```diff
@@ -586,12 +599,17 @@ private void fillLicenseList(XWPFDocument document, Collection<LicenseInfoParsin
         XWPFRun licenseHeaderRun = document.createParagraph().createRun();
         addFormattedText(licenseHeaderRun, "License texts", FONT_SIZE + 2, true);
         addNewLines(document, 0);
-
+        int licenseCount = 1;
+        StringBuilder licenseNameWithCount = new StringBuilder();
         for (LicenseNameWithText licenseNameWithText : licenseNameWithTexts) {
             XWPFParagraph licenseParagraph = document.createParagraph();
             licenseParagraph.setStyle(STYLE_HEADING);
             String licenseName = licenseNameWithText.isSetLicenseName() ? licenseNameWithText.getLicenseName() : UNKNOWN_LICENSE_NAME;
-            addBookmark(licenseParagraph, licenseName, licenseName);
+            licenseNameWithCount.append(licenseCount).append(": ");
```

Comment:
Please consider refractoring it to a single line.

Reply:
Done.